### ISSUE TITLE
ModuleNotFoundError: No module named 'core_tools.startup'

### ISSUE THREAD

**Issue by haroldmeerwaldt** [stephanlphilips/core_tools#12]
Hi @sldesnoo-Delft,

After the latest update (to 1.4.0), we get the following ModuleNotFoundError:
```python
src\lls\utils\core_tools\__init__.py:12: in <module>
    from core_tools.drivers.hardware.virtual_gate_matrix import VirtualGateMatrix, VirtualGateMatrixView
env\lib\site-packages\core_tools\__init__.py:2: in <module>
    from core_tools.startup.all import configure
E   ModuleNotFoundError: No module named 'core_tools.startup'
```

Could you have a look at this?

Sincerely, Harold

**Comment by sldesnoo-Delft** [comment]
__init__.py was missing. Should be fixed with release 1.4.1

**Comment by haroldmeerwaldt** [comment]
Solved it. Thanks for the fast response!
